base/libgit2/blob.jl

```diff
@@ -8,10 +8,34 @@ function Base.length(blob::GitBlob)
     return ccall((:git_blob_rawsize, :libgit2), Int64, (Ptr{Void},), blob.ptr)
 end
 
+"""
+Use a heuristic to guess if a file is binary: searching for NULL bytes and
```

Comment:
I just figured that was standard practice for docstrings since it's what I've seen most often. Up to you though, of course--just a suggestion.